### PR TITLE
PSP-9151 : Expand notes tab to improve readability

### PR DESCRIPTION
**Pull request by eddherrera**
image

### CODE REVIEW

File: source/frontend/src/features/mapSideBar/research/tabs/ResearchTabsContainer.test.tsx [hunk 1]
```diff
@@ -82,7 +80,7 @@ describe('ResearchFileTabs component', () => {
   });
 
   it('notes tab can be changed to', async () => {
-    const { getAllByText } = setup(
+    const { getAllByText, container } = setup(
```
Comment (devinleighsmith) [inline source/frontend/src/features/mapSideBar/research/tabs/ResearchTabsContainer.test.tsx:83]: why is this necessary?

**Review by devinleighsmith — APPROVED** [review]

File: source/frontend/src/features/mapSideBar/research/tabs/ResearchTabsContainer.test.tsx (hunk 1 above)
Comment (eddherrera) [reply source/frontend/src/features/mapSideBar/research/tabs/ResearchTabsContainer.test.tsx:83]: removed
